### PR TITLE
CLC-6338 Code formatted

### PR DESCRIPTION
**Pull request by sethlu**
- Fixed button start tags (w3c spec)
- Removed ending slash in void elements
- Better consistency in HTML (now not conformed to XHTML)
- Indentation not addressed

### CODE REVIEW

File: src/assets/javascripts/angular/directives/youtubeDirective.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ angular.module('calcentral.directives').directive('ccYoutubeDirective', function
 
         // Templates for the player
         var templates = {
-          image: '<button id="cc-youtube-image-placeholder" class="cc-youtube-image-placeholder" tabindex="0"><span class="cc-visuallyhidden">Play video' + title + '</span><img data-ng-src="' + imageUrl + '" alt=""></img><div class="cc-youtube-thumbnail-button"></div></button>',
```
Comment (raydavis) [inline src/assets/javascripts/angular/directives/youtubeDirective.js]: Ouch. Good catch.

File: src/assets/templates/dashboard_task_editor.html [hunk 2]
```diff
@@ -56,18 +56,17 @@
...
           <span data-ng-show="editorEnabled">Save</span>
       </button>
       <button class="cc-button" type="reset"
         data-ng-show="!editorEnabled"
-        data-ng-click="addEditTask.title='';toggleAddTask()" />Cancel
+        data-ng-click="addEditTask.title='';toggleAddTask()">Cancel
       </button>
       <button class="cc-button" type="button"
         data-ng-show="editorEnabled"
-        data-ng-click="disableEditor()" />Cancel
```
Comment (raydavis) [inline src/assets/templates/dashboard_task_editor.html]: Another good one.

File: src/assets/templates/academics_classinfo.html [hunk 3]
```diff
@@ -166,15 +166,13 @@ <h3 class="cc-book-title" data-ng-bind="sectionBookList.sectionLabel" data-ng-if
          data-seats-available="selectedCourse.waitlistLimit"
          data-show-position="true"
          data-students="waitlistedStudents"
-         data-title="Wait List"
-    ></div>
+         data-title="Wait List"></div>
```
Comment (raydavis) [inline src/assets/templates/academics_classinfo.html]: Personally I find the original indentation style clearer here.

**Comment by raydavis** [comment]
Apologies for the very tardy response, @sethlu . As you might have picked up from some of the other GitHub activity, our team is currently slammed pretty hard by already scheduled high-priority tasks.

Your attention is greatly appreciated here. We've built JS and CSS checks into continuous integration, but the last time we looked we were unable to find an HTML style-checker which could deal with AngularJS template fragments; of course, Ruby-embedded HTML is even harder to check. And as the old saying goes, untested code is broken code. 

If I can attempt to summarize your PR, it's a mix of
1. finding and fixing some unambiguous bugs (prematurely or incorrectly closed tags),
2. getting rid of deprecated and inconsistent formats (the unnecessary "/"s in img, hr, etc.), and
3. some things which look like battling ideas of code style (how to handle indentation at the end of a multi-line tag).
I'd love to merge the first two types of change but leave out the third type, at least pending further discussion with the team.

Unfortunately with this kind of long-pending touch-a-lot-of-places PR, merge conflicts are likely, and it looks like some have already crept in. If you have time to resolve the conflicts and remove the more arguable stylistic changes, I'd love to get these improvements into the product.

I totally understand that you might not have time right now though, and if that's the case, we can just schedule a task to incorporate your fixes at a later date.

Again, many thanks for devoting some time to CalCentral.

**Comment by raydavis** [comment]
I've linked this PR to the CalCentral task CLC-6338.

**Comment by sethlu** [comment]
@raydavis Thanks for the response! I really appreciate your team's effort on maintaining this migratory project. For most times I use PHP for server-side page generation, and keeping a consistency throughout the many fragments is agreeably troublesome.
With this PR, I'll take your advice and keeping the changes mentioned by the first two bullet points. It may take me some time, but coming very soon, to reset the previous commit and breakdown some parts of the changes so you may squash/merge them easier.

**Comment by raydavis** [comment]
Thank you for your patience and for the offer of further assistance, @sethlu ! I know the life of a student isn't always predictable, so no worries if something blocks you. What you've pointed out already is a big help.

By the way, we're always interested in hearing ideas on the features and design of CalCentral, too, so pass them along if you have them. Currently our sprints are pretty well flooded by SIS Project transitions, but we eventually expect to get back to other functional and technical improvements.

**Comment by sethlu** [comment]
@raydavis sure, please check the most recent commits of the changes broken down. I've removed the previous concerns over code indenting inconsistencies, and additionally corrected some pairing issues in some page files. Let me know if there be more that I may help with.

**Comment by raydavis** [comment]
Fantastic job, @sethlu . Thank you again for your assistance.

**Pull request merged**